cache/config.go

```diff
@@ -29,6 +29,8 @@ type Config struct {
 	InitialBackoff time.Duration
 	// MaxBackoff caps the exponential back-off between successive upstream watch retries.
 	MaxBackoff time.Duration
+	// InitialLoadTimeout is the timeout applied to the first Get() used to bootstrap the cache.
+	InitialLoadTimeout time.Duration
```

Comment:
```suggestion
	// GetTimeout is the timeout applied to the Get() used to bootstrap the cache.
	GetTimeout time.Duration
```
Don't invent new terms.